Dockerfile

```diff
@@ -26,8 +26,9 @@ RUN apt-get update && \
 RUN ls -l
 RUN mkdir /scamper-src
 ADD ./vendor/scamper/ /scamper-src
-RUN chmod +x /scamper-src/scamper-cvs-20190916/configure
-WORKDIR /scamper-src/scamper-cvs-20190916/
+RUN tar xvzf  /scamper-src/scamper-cvs-20191102a.tar.gz -C /scamper-src/
+RUN chmod +x /scamper-src/scamper-cvs-20191102a/configure
```

Comment:
It looks like the right directory is `/scamper-src/scamper-cvs-20191102a/scamper/configure`